### PR TITLE
Changes to conftest.py for translate tests

### PR DESCRIPTION
**Pull request by fmalatino**
**Description**
The `conftest.py` file for the translate tests has been changed to match centralized test structure in `NDSL`.

**How Has This Been Tested?**
Tested with current translate test structure

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] New check tests, if applicable, are included
- [x] Targeted model, if this change was triggered by a model need/shortcoming

### CODE REVIEW

**Comment by bensonr** [comment]
@fmalatino - can you change this from a draft if it is ready to be evaluated?

File: setup.py [hunk 1]
```diff
@@ -14,8 +14,10 @@
     "xarray",
 ]
 
-test_requirements = ["pytest", "pytest-subtests", "serialbox"]
-ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@2024.04.00"]
+test_requirements = ["pytest", "pytest-subtests", "serialbox", "coverage"]
+ndsl_requirements = [
+    "ndsl @ git+https://github.com/fmalatino/NDSL.git@fix/computegridoption"
```
Comment (bensonr) [inline setup.py]: We should get your ndsl branch into `main` and/or released to omit this special reference.

**Review by bensonr — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (fmalatino) [reply setup.py]: I still have this on the test branch and as a draft, as there is a PR in NDSL ([PR 53](https://github.com/NOAA-GFDL/NDSL/pull/53) ) that is a hotfix that is needed before the next release. Once those are merged I will change the branch to point the new release branch.

**Review by bensonr — DISMISSED** [review]
based on what I know, this looks good to me

File: .github/workflows/translate.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+name: "Translate test"
```
Comment (mlee03) [inline .github/workflows/translate.yml]: this file is not needed

Comment (fmalatino) [reply .github/workflows/translate.yml]: Removed.

**Review by mlee03 — APPROVED** [review]

**Review by bensonr — APPROVED** [review]

**Pull request merged**